ept/ept.py

```diff
@@ -59,7 +88,7 @@ def data(self):
         return o
 
     async def adata(self):
-        limit = 10
+        limit = 100
```

Comment:
Was there bad consequences of raising it this high?

Reply:
I was actually going to start a separate issue for this. For the same query run multiple times, I could see consistent GET requests (local server for ept data) in the server logs for 23 laz files. But the task pool would randomly return only 10-14 results. When I upped the semaphore value to 100, I then got all 23 files back. The increase was a hack to get things working.